### PR TITLE
Add basic WebExtension example (Manifest V3)

### PR DESCRIPTION
**Pull request by flosse**

### CODE REVIEW

File: examples/basic-web-extension/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[workspace]
+members = [
+  "background-script"
```
Comment (rvolosatovs) [inline examples/basic-web-extension/Cargo.toml:3]: What is this for?

File: examples/basic-web-extension/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+]
+
+[patch.crates-io]
+web-extensions-sys = { path = "../../" }
```
Comment (rvolosatovs) [inline examples/basic-web-extension/Cargo.toml:7]: Why not import directly via `dependencies`?

Comment (flosse) [reply examples/basic-web-extension/Cargo.toml:7]: I'll extend the example with a foreground script and popup example. Then all would need to point to a relative path.
With this approach you can even comment this line out and run the example against the published crate on crates.io

**Comment by flosse** [comment]
I maked this now as draft PR, so you can wait until the example is a bit more extended.

**Pull request merged**